### PR TITLE
add OZW status enum

### PR DESCRIPTION
**Pull request by marcelveldt**
as requested in the core PR

### CODE REVIEW

**Comment by marcelveldt** [comment]
@cgarwood do you have time to merge this in and publish new version of the lib?
Trying to get a grasp of the latest small issues.

File: openzwavemqtt/const.py [hunk 1]
```diff
@@ -33,6 +33,34 @@
...
 
+class OpenZWaveStatus(Enum):
+    """Enum with all Status strings for the OZW Daemon."""
+
+    # https://github.com/OpenZWave/qt-openzwave/blob/master/docs/MQTT.md#mqtt-client-status
+
+    OFFLINE = "Offline"
+    STARTING = "starting"
+    STARTED = "started"
+    STOPPED = "stopped"
```
Comment (MartinHjelmare) [inline openzwavemqtt/const.py:44]: In the docs it says `stoped`. Is that just a typo in the docs or is it the api that has a typo?

Comment (marcelveldt) [reply openzwavemqtt/const.py:44]: I assumed the api

Comment (MartinHjelmare) [reply openzwavemqtt/const.py:44]: So then we should change to `"stoped"`?

Comment (marcelveldt) [reply openzwavemqtt/const.py:44]: No, stoped was the typo. API is stopped.

**Pull request merged**